formats/properties/commonMain/src/kotlinx/serialization/properties/Properties.kt

```diff
@@ -146,6 +204,14 @@ public fun Properties(module: SerializersModule): Properties = PropertiesImpl(mo
 public inline fun <reified T> Properties.encodeToMap(value: T): Map<String, Any> =
     encodeToMap(serializersModule.serializer(), value)
 
+/**
+ * Encodes properties from given [value] to a map using serializer for reified type [T] and returns this map.
```

Comment:
This documentation does not mention strings or whatsoever, so it looks like it was based on the `encodeToMap` function, not the `encodeToStringMap`

Reply:
Applied suggestion.